assets/pages/contributions-thankyou/oneoffContributionsThankyou.jsx

```diff
@@ -59,6 +59,9 @@ const content = (
         </InfoSection>
       </div>
     </section>
+    <section className="gu-content-filler">
```

Comment:
If this is going to be necessary on all pages, can we avoid the duplication by putting it in the SimpleFooter template?